server/db.js

```diff
@@ -0,0 +1,65 @@
+/*========== EXTERNAL MODULES ==========*/
+const {Client} = require('pg');
+require('dotenv').config();
+
+/*========== INTERNAL MODULES ==========*/
+const {USER, PASSWORD} = require('../.env');
+
+
+const client = new Client({
+  user: {USER},
+  password: {PASSWORD}.
+  database: 'products',
+})
+
+client.connect((err) => {
+  if (err) throw err;
+})
+
+
+/*========== DATABASE ==========*/
```

Comment:
You may need to move this table creation code to a .sql file.  Worth investigating.